blog/2024-01-24-pants-2-19/index.mdx

```diff
@@ -0,0 +1,79 @@
+  /
+  [Unsplash](https://unsplash.com/photos/mountains-covered-with-snow-d3pTF3r_hwY?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)
+</CaptionedImg>
+
+{/* truncate */}
+
+We're pleased to announce Pants 2.19.0, the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.19.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/docs/getting-started).
+
+_Highlights in 2.19 include_:
+
+- New backends: `semgrep` and `openapi-format`
+- Test retries for flaky Python tests
+- `buildx` support in the Docker backend
+- Using `parametrize` to set multiple fields in tandem
+- And lots of smaller features, bugfixes, and general improvements
+
+Keep reading to see more, or check the [full changelog](https://github.com/pantsbuild/pants/tree/main/src/python/pants/notes/2.19.x.md). Pants is an open-source project, and the below changes are all contributed by our community. If you want to see something more in the next changelog, join us on [GitHub](https://github.com/pantsbuild/pants) and become a contributor.
+
+We also offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/sponsorship), as well as individual sponsorships via [GitHub](https://github.com/sponsors/pantsbuild). These help pay for the ongoing development and hosting costs, and are managed by the Pants Build non-profit organization.
+
+### New backends and subsystems
+
+In this release, we welcome two new backends and one new subsystem.
+
+[The new `openapi_format` backend](https://www.pantsbuild.org/2.19/reference/subsystems/openapi-format) supports running [openapi-format](https://github.com/thim81/openapi-format) to format `openapi_...` targets. You can enable this by adding `pants.backend.experimental.openapi.lint.openapi_format` to the list of active backends.
+
+[The new `semgrep` backend](https://www.pantsbuild.org/2.19/reference/subsystems/semgrep) supports running [Semgrep](https://semgrep.dev) as a local linter on all files that Pants knows about (any target with a `source` or `sources` field). Configuration files are automatically discovered. You can enable this by adding `pants.backend.experimental.tools.semgrep` to the list of active backends.
+
+[The new `[system-binaries]` subsystem](https://www.pantsbuild.org/2.19/reference/subsystems/system-binaries) allows controlling where Pants searches for binaries like `bash`, `zip`, `tar` that is uses as part of normal operation. Set [the `system_binaries_paths`](https://www.pantsbuild.org/2.19/reference/subsystems/system-binaries#system_binary_paths) as appropriate, potentially using the `<PATH>` special value. This can also be set on specific [environments](https://www.pantsbuild.org/2.19/docs/using-pants/environments) using the `system_binaries_system_binary_paths` field.
```

Comment:
```suggestion
[The new `[system-binaries]` subsystem](https://www.pantsbuild.org/2.19/reference/subsystems/system-binaries) allows controlling where Pants searches for binaries like `bash`, `zip`, `tar` that it uses as part of normal operation. See [the `system_binaries_paths`](https://www.pantsbuild.org/2.19/reference/subsystems/system-binaries#system_binary_paths) as appropriate, potentially using the `<PATH>` special value. This can also be set on specific [environments](https://www.pantsbuild.org/2.19/docs/using-pants/environments) using the `system_binaries_system_binary_paths` field.
```

Reply:
I rephrased it to be a bit more clear, what do you think?